AmplifyPlugins/Auth/AWSAuthPlugin/Support/Constants/AuthPluginErrorConstants.swift

```diff
@@ -27,6 +27,14 @@ struct AuthPluginErrorConstants {
     static let mobileClientInitializeError: AuthPluginErrorString = (
         "Unable to initialize the underlying AWSMobileClient",
         "Make sure that the necessary configuration are present in the configuration file")
+
+    static let hostedUISecurityFailedError: AuthPluginErrorString = (
+        "Found invalid parameter while parsing the webUI signIn",
+        "Make sure that the signIn url is not tampered. Contact AWS if this issue happens.")
```

Comment:
This is a weird message on several levels. 1) The grammar needs a bit of help; 2) We're advising customers to contact AWS, but providing no contact information or path forward; 3) We're saying this is a security error, but not giving it any particular urgency other than this message.

What does condition does this actually map to? How should we be advising customers to proceed if they encounter this?

Reply:
As discussed:
ErrorDescription = "Found invalid parameter while parsing the webUI redirect URL"
RecoveryDescription = "Make sure that the signIn URL has not been modified during the signIn flow."